Formula/mpfi.rb

```diff
@@ -16,15 +16,20 @@ class Mpfi < Formula
     sha256 cellar: :any_skip_relocation, x86_64_linux:  "5bc5ac4e71fcaf3e978ef82410d544f274e4652899bcb06bf2685d9df6a63bed"
   end
 
-  # Formula does not build, https://gforge.inria.fr/tracker/index.php?func=detail&aid=21721&group_id=157&atid=709
-  # and upstream is not actively maintaining (last commit was on 2019-08-01)
-  deprecate! date: "2021-08-15", because: :unmaintained
+  head do
+    url "https://gitlab.inria.fr/mpfi/mpfi.git", branch: "master"
+
+    depends_on "autoconf" => :build
+    depends_on "automake" => :build
+    depends_on "libtool" => :build
+  end
 
   depends_on "gmp"
   depends_on "mpfr"
 
   def install
-    system "./configure", "--disable-dependency-tracking", "--prefix=#{prefix}"
+    system "./autogen.sh" if build.head?
+    system "./configure", *std_configure_args
     system "make"
     system "make", "check"
```

Comment:
We could disable the `make check` to ignore test failures. Not sure what is exact issue causing segfaults on 2 tests and 1-2 other test failures. Remaining 150+ tests all pass.